modules/ossm-security-cert-manage.adoc

```diff
@@ -0,0 +1,130 @@
+//
+// * service_mesh/service_mesh_user_guide/ossm-security.adoc
+
+[id="ossm-cert-manage"]
+= Adding an external certificate authority key and certificate
+
+By default, {ProductName} generates self-signed root certificate and key, and uses them to sign the workload certificates. You can also use the operator-specified certificate and key to sign workload certificates, with operator-specified root certificate. This task demonstrates an example to plug certificates and key into {ProductShortName}.
+
+.Prerequisites
+
+* You must have installed {ProductName} with mutual TLS enabled to configure certificates.
+* You must use the certificates from link:https://github.com/maistra/istio/tree/maistra-2.0/samples/certs[Maistra repo]. 
+* You must deploy the Bookinfo sample application to verify the results with these instructions.
+
+[id="ossm-cert-manage-add-cert-key"]
+== Adding an existing certificate and key
+
+To use an existing signing (CA) certificate and key, you must create a chain of trust file that includes the CA certificate, key, and root certificate. You must use the following exact file names for each of the corresponding certificates. The CA certificate is called `ca-cert.pem`, the key is `ca-key.pem`, and the root certificate, which signs `ca-cert.pem`, is called `root-cert.pem`. If your workload uses intermediate certificates, you must specify them in a `cert-chain.pem` file. 
+
+Add the certificates to {ProductShortName} by following these steps. Save the certificates from the link:https://github.com/maistra/istio/tree/maistra-2.0/samples/certs[Maistra repo] locally and replace `<path>` with the path to your certificates.
+
+1. Create a secret `cacert` that includes the input files `ca-cert.pem`, `ca-key.pem`, `root-cert.pem` and `cert-chain.pem`.
++
+----
+$ oc create secret generic cacerts -n istio-system --from-file=<path>/ca-cert.pem \
+    --from-file=<path>/ca-key.pem --from-file=<path>/root-cert.pem \
+    --from-file=<path>/cert-chain.pem
+----
++
+2. In the `ServiceMeshControlPlane` resource set `global.mtls.enabled` to `true` and `security.selfSigned` set to `false`. {ProductShortName} reads the certificates and key from the secret-mount files.
++
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshControlPlane
+spec:
+  istio:
+    global:
+      mtls:
+        enabled: true
+    security:
+      selfSigned: false
+----
++
+3. To make sure the workloads add the new certificates promptly, delete the secrets generated by {ProductShortName}, named `istio.*`. In this example, `istio.default`. {ProductShortName} issues new certificates for the workloads.
++
+----
+$ oc delete secret istio.default
+----
+
+[id="ossm-cert-manage-verify-cert"]
+== Verifying your certificates
+
+Use the Bookinfo sample application to verify your certificates are mounted correctly.
+
+1. Retrieve the mounted certificates. Verify the certificates mounted on the pod. Store the pod name in the variable `RATINGSPOD`.
```

Comment:
There are three actions in this step, again, I feel like there are missing commands or steps somewhere?  And we're retrieving the certificates in the next step?  And verifying in the last step.  (Is the text in step one really an overview for the whole procedure or what you're doing in step one?)

Reply:
done